docker-compose.selenium.yml

```diff
@@ -0,0 +1,113 @@
+version: '3'
+
+services:
+  tests:
+    build:
+      context: ./
+      dockerfile: SeleniumDockerfile
```

Comment:
Might be good to rename this Dockerfile.selenium so its consistent with the docker-compose files and repo file ordering.